### PR TITLE
Succinct error contexts

### PR DESCRIPTION
**Pull request by codygibb**
Recommend keeping error contexts succinct.

### CODE REVIEW

**Comment by codygibb** [comment]
SGTM. I'll expand on this a bit more.

File: style.md [hunk 1]
```diff
@@ -729,8 +729,51 @@ There are three main options for propagating errors if a call fails:
...
-error such as "connection refused", you get more useful errors such as "failed to
-call service foo: connection refused".
+error such as "connection refused", you get more useful errors such as
+"call service foo: connection refused".
+
+When adding context to returned errors, keep the context succinct by avoiding
+phrases like "failed to", which state the obvious and pile up as the error
+percolates up through the stack:
+
+<table>
```
Comment (prashantv) [inline style.md]: this example is resulting in scrollbars for the code snippets:
https://github.com/uber-go/guide/blob/4f628ff1b1130b013bb2ab46cf0b585fa6cb2d5a/style.md#error-wrapping

can you play with the example to still make the point that "failed to" gets repetitive, but also reducing the line length so it fits in the default github width?

Comment (codygibb) [reply style.md]: I was irked by it as well... Yeah I'll get this looking nice.

Comment (codygibb) [reply style.md]: I placed a somewhat questionable line break in the code. Without it we have a tiny bit of scroll bar. Lmk if you'd prefer a little scroll over the line break.

Comment (prashantv) [reply style.md]: lgtm, I think I'd prefer to avoid the scrollbar.

**Pull request merged**